RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -794,34 +794,73 @@ public static void loadPropertiesFile() {
 
         applicationProperties = new AppConfig(file);
 
-        getApplicationProperty("useRoddyVersion", LibrariesFactory.PLUGIN_VERSION_CURRENT); // Load some default properties
-        getApplicationProperty("usePluginVersion");
-        getApplicationProperty("pluginDirectories");
-        getApplicationProperty("configurationDirectories");
+        getOrSetApplicationProperty("useRoddyVersion", LibrariesFactory.PLUGIN_VERSION_CURRENT); // Load some default properties
+        getOrSetApplicationProperty("usePluginVersion");
+        getOrSetApplicationProperty("pluginDirectories");
+        getOrSetApplicationProperty("configurationDirectories");
     }
 
     public static AppConfig getApplicationConfiguration() {
         return applicationProperties;
     }
 
-    public static String getApplicationProperty(String pName) {
-        return getApplicationProperty(pName, "");
+    public static String getOrSetApplicationProperty(String pName) {
```

Comment:
We might be able to get rid of the runmode prefix in the ini files:
- Parse over all values in the loaded ini and remove the CLI. prefix (or store a copy without the prefix).
- Get rid of the runmode part in all the method headers.
- Ignore prefixes other than CLI. when loading inis


Should make ini files clearer and still backward compatible.